nixos/modules/services/monitoring/nagios.nix

```diff
@@ -149,9 +145,11 @@ in
       description = "Nagios user ";
       uid         = config.ids.uids.nagios;
       home        = nagiosState;
-      createHome  = true;
+      group       = "nagios";
     };
 
+    users.groups.nagios = { };
```

Comment:
Don't we need to set the gid here (like the uid above)? Otherwise :+1: from here

Reply:
@peterhoeg While some modules need a gid/uid which will be the same across machines I'm not sure that `nagios` requires this. I can't think of a case where this would be required... but if anyone else can please speak up.

Reply:
@peterhoeg Is this an acceptable answer?

Reply:
Yes, sorry about the silence.

Reply:
No problem. Thanks @peterhoeg!